src/lib/transition_frontier/full_frontier/full_frontier.mli

```diff
@@ -0,0 +1,30 @@
+open Coda_base
+open Frontier_base
+
+include Frontier_intf.S
+
```

Comment:
I documented the names and their purposes in the description of this PR. If you don't like `Full_frontier`, what do you like? I think it makes sense because the persistent frontier has `Lite` node representations, where as the in memory one has `Full` node representations (mapping logically over to the node representations in diffs).